Solutions/Marain.TenantManagement.Specs/Data/ServiceManifests/FooBarServiceManifest.json

```diff
@@ -1,6 +1,7 @@
 {
   "$schema": "./ServiceManifestSchema.json",
   "contentType": "application/vnd.marain.tenancy.servicemanifests.servicemanifest",
+  "wellKnownTenantGuid": "ff74a61e-3060-40d3-bfec-60dfe12febac", // 3633754ac4c9be44b55bfe791b1780f11ea674ff6030d340bfec60dfe12febac
```

Comment:
Why does this one get the encoded ID in the comment but the rest don't? (Since GitHub slaps our wrists because IT'S NOT JSON, I'd be inclined to remove it from here, unless you want to rename these all to be `.jsonc`.)

Reply:
Basically I added the ones I care about. And I care about the Id of this one as it needs to be included in the dependencies for the `OperationsServiceManifest.json` in the same folder.